packages/flutter/lib/src/widgets/debug.dart

```diff
@@ -114,8 +114,17 @@ bool debugPrintGlobalKeyedWidgetLifecycle = false;
 ///  * [debugProfileLayoutsEnabled], which does something similar for layout,
 ///    and [debugPrintLayouts], its console equivalent.
 ///  * [debugProfilePaintsEnabled], which does something similar for painting.
+///  * [debugProfileBuildsEnabledUserWidgets], adds events for user generated
+///    [Widget] build.
 bool debugProfileBuildsEnabled = false;
 
+/// Adds [Timeline] events for every user created [Widget] built.
+///
+/// See also:
+///  * [debugProfileBuildsEnabled], functions similarly but shows events for
```

Comment:
you'll need a blank line after the "See also:"

Reply:
done